include/range/v3/action/insert.hpp

```diff
@@ -76,23 +76,76 @@ namespace ranges
                 return unwrap_reference(cont).insert(p, n, std::forward<T>(t));
             }
 
+            namespace impl {
```

Comment:
Also, use [the Allman brace style](https://en.wikipedia.org/wiki/Indent_style#Allman_style) please.